reference-implementation/lib/writable-stream.js

```diff
@@ -332,15 +315,16 @@ function WritableStreamHandleAbortRequestIfPending(stream) {
     return;
   }
 
-  WritableStreamFinishAbort(stream);
-
   const abortRequest = stream._pendingAbortRequest;
   stream._pendingAbortRequest = undefined;
   const promise = WritableStreamDefaultControllerAbort(stream._writableStreamController, abortRequest._reason);
   promise.then(
     abortRequest._resolve,
     abortRequest._reject
   );
+  stream._backpressure = false;
```

Comment:
stream._backpressure is normally updated by WritableStreamUpdateBackpressure, so this feels like a bit of a rough edge to me too.